Libs/vtkITK/vtkITKArchetypeDiffusionTensorImageReaderFile.cxx

```diff
@@ -96,7 +96,7 @@ void vtkITKExecuteDataFromFileDiffusionTensor3D(
   for ( it.GoToBegin(); !it.IsAtEnd() ; ++it )
   {
     const itk::Index<3u> index = it.GetIndex();
-    vtkIdType position = data->FindPoint(index[0], index[1], index[2]);
+    vtkIdType position = data->vtkDataSet::FindPoint(index[0], index[1], index[2]);
```

Comment:
The removal of `virtual vtkIdType FindPoint(double x, double y, double z)` from `vtkImageData` seems to have been introduced in 92fc5b2670 (`vtkImageData: Use Implicit structures and check visibility`, 2023-05-18) by @spyridon97

The following pull request is intended to address this. See https://gitlab.kitware.com/vtk/vtk/-/merge_requests/11935

Reply:
Corresponding changes have been backported to https://github.com/Slicer/VTK/commits/slicer-v9.4.1-2024-12-26-6b6b89ee5/